Web/www/index.html

```diff
@@ -0,0 +1,194 @@
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">I</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">D</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+
+						<br><br>
+						<p class="text--bold">Roll : </p>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">P</label>
+						<input id="roll_p" type="text" class="card_input">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">I</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">D</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+
+						<br><br>
+						<p class="text--bold">Yaw : </p>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">P</label>
+						<input type="text" class="card_input">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">I</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">D</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+
+						<br><br>
+						<div style="text-align: center;">
+							<button class="text--bold" type="submit">Submit</button>
+						</div> -->
+					</div>
+				</div>
+			</section>
+		</main>
+		<script>
+			var pid_card = document.getElementById("pid_card");
+			var names = ["Pitch", "Roll", "Yaw"];
+			fetch('pid.json')
+				.then(response => response.json())
+				.then(data => appendPID(data));
+			
+			function appendPID(data){
+				console.log(data);
+				// Run for pitch, roll and yaw
+				for (var i = 0; i < data.length; i++) {
+					var _br = document.createElement("br");
```

Comment:
```javascript
const propertyMarkup = `
  <div class="property-container">
    <p>${property.name}</p>
    ...
  </div>
`;
```
This is a decent way to do it if you can't use that `pyhtml` templating language thing. I'll give more details on that if you can't use the `.pyhtml` but I would say that it is much cleaner to do it that way.